interpreter/exec/eval.ml

```diff
@@ -57,19 +59,27 @@ and admin_instr' =
   | Label of int32 * instr list * code
   | Frame of int32 * frame * code
 
+type suspend_signal =
+    No_signal
+    (* memory, cell index, target timestamp *)
+  | Wait_signal of memory_inst * Memory.address * float
```

Comment:
The intention is that in the "long term" this would be fed by `gettimeofday`, which returns a float directly - does it make sense to use `F64.t` here?

Reply:
Good question, it should correspond to what we using on paper. Is that a real number?

Reply:
The paper's timestamps would be naturally be rationals. I guess I could use Zarith instead, but this would be an extra dependency.

Reply:
Yeah, in that case float probably is the closest match.